oauth2_provider/migrations/0003_authorization_grant_type.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 1.11.4 on 2017-09-03 16:32
+from django.db import migrations, models
```

Comment:
remove this migration too. after you are done with all models related changes and association, regenerate new migrations based on the 2 existing migations from master. better use django 2.2

Reply:
Done